src/common/Pagination/styled.js

```diff
@@ -0,0 +1,97 @@
+import { styled } from "styled-components";
+import { ReactComponent as IconRight } from "../../images/vectorRight.svg";
+import { ReactComponent as IconLeft } from "../../images/vectorLeft.svg";
+
+export const Wrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: 400px 0 103px 0; // 400px na potrzeby wizualizacji :-)
+  gap: 12px;
+
+  @media (max-width: ${({ theme }) => theme.common.breakpoint}) {
+    gap: 8px;
+    margin: 16.5px 0 31.5px 0;
+  }
+`;
+
+export const Button = styled.button`
+  padding: 8px 16px;
+  background-color: ${({ theme }) => theme.colors.pattensBlue};
+  border: none;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  gap: 6px;
+  transition: 1s;
```

Comment:
I think it might be a bit better if the transition is a bit shorter here 😎
Also I'd consider using `cursor: pointer; ` as an additional interaction with buttons 👌

Reply:
ok